### DISCUSSION TITLE
Fetching location in background on iOS

### DISCUSSION THREAD

**Discussion by simplylizz** [Lyokone/flutterlocation#823] [Q&A]
Hello,

I'm trying to get background updates of location and send them to my server, so I used a code like this:
```
setLocationSettings(
    accuracy: LocationAccuracy.high,
    smallestDisplacement: 50.0,
  );
  onLocationChanged(inBackground: true).listen(sendGeo);
```

I deployed an app to my phone to test it and now I see that it usually sends updates only within short time after I close it. After that it barely sends any info to the server. E.g. 2 last days it was just 1 update per day, today when I opened the app it was sending updates within ~30 mins after closure, but since then it stopped and there is nothing happening within ~2 hours.

Also, I don't need to get these updates too often, once per 5-10 mins would be ideally. Currently I check time of the last update in my `sendGeo` function, but before I was also experimenting with `interval` settings, can't say that I see much difference.

My question is: is this really the best I can get with iOS?

P.S. Just in case, I'm running the app on iPhone 13 Pro, iOS 16.3.1.

**Comment by H7mosh** [comment]
Hi . did you solve that problem or not yet ?? 
if you solve it . can you tell me how did you solve it ?

**Reply by simplylizz** [reply]
Nope
